### DISCUSSION TITLE
anyone have a good suggestion for a simple webchat solution

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [phalcon/cphalcon#16579] [General]
I am just looking for tiny / simple solution that connects a website visitor with an xmpp client. nothing fancy with audio/video/files, just a bit of writing in an overlay/pop-out window. I already have an xmpp server and an xmpp client on the desktop.

**Comment by victor-vb** [comment]
Hi, maybe you need a framework like workman to inherit from phalcon. Since phlacon and swoole are not compatible, we can only recommend a framework like workman that does not rely on other libraries, and the performance of workman is also very powerful.
